addons/packages/antrea/1.5.2/bundle/config/schema.yaml

```diff
@@ -0,0 +1,96 @@
+      portRange: ""
+    #@schema/desc "AntreaProxy related configuration options."
+    antreaProxy:
+      #@schema/desc "ProxyAll tells antrea-agent to proxy all Service traffic."
+      proxyAll: false
+      #@schema/desc "Specifies the host IPv4/IPv6 addresses for NodePort."
+      nodePortAddresses:
+       - ""
+      #@schema/desc "List of Services which should be ignored by AntreaProxy."
+      skipServices:
+        - ""
+      #@schema/desc "Load-balance traffic destined to the External IPs of LoadBalancer services."
+      proxyLoadBalancerIPs: false
+    #@schema/desc "Provide the address of Kubernetes apiserver."
+    #@schema/nullable
+    kubeAPIServerOverride: ""
+    #@schema/desc "The name of the interface on Node which is used for tunneling or routing the traffic."
+    #@schema/nullable
+    transportInterface: ""
+    #@schema/desc "The network CIDRs of the interface on Node which is used for tunneling or routing the traffic."
+    transportInterfaceCIDRs:
+      - ""
+    #@schema/desc "The names of the interfaces on Nodes that are used to forward multicast traffic."
+    multicastInterfaces:
+      - ""
+    #@schema/desc "Tunnel protocols used for encapsulating traffic across Nodes. One of the following options => geneve, vxlan, gre, stt"
+    tunnelType: none
+    #@schema/desc "Determines how tunnel traffic is encrypted. One of the following options => none, ipsec, wireGuard"
+    trafficEncryptionMode: none
+    #@schema/desc "WireGuard related configurations."
+    wireGuard:
+      #@schema/desc "The port for WireGuard to receive traffic."
+      port: 51820
+    #@schema/desc "Enable usage reporting (telemetry) to VMware."
+    enableUsageReporting: false
+    #@schema/desc "ClusterIP CIDR range for IPv4 Services"
+    serviceCIDR: 10.96.0.0/12
+    #@schema/desc "ClusterIP CIDR range for IPv6 Services"
+    #@schema/nullable
+    serviceCIDRv6: ""
+    #@schema/desc "The traffic encapsulation mode. One of the following options => encap, noEncap, hybrid, networkPolicyOnly"
+    trafficEncapMode: encap
+    #@schema/desc "Flag to enable/disable SNAT for the egress traffic from a Pod to the external network"
+    noSNAT: false
+    #@schema/desc "Disable UDP tunnel offload feature on default NIC"
+    disableUdpTunnelOffload: false
+    #@schema/desc "Default MTU to use for the host gateway interface and the network interface of each Pod"
+    #@schema/nullable
+    defaultMTU: ""
+    #@schema/desc "List of allowed cipher suites. If omitted, the default Go Cipher Suites will be used"
+    tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384
+    #@schema/desc "FeatureGates is a map of feature names to flags that enable or disable experimental features"
+    featureGates:
+      #@schema/desc "Flag to enable/disable antrea proxy"
+      AntreaProxy: false
```

Comment:
I'm not sure about the relationship between the default values in schema.yaml and values.yaml. In values.yaml the `AntreaProxy` is true.